src/build/build.ts

```diff
@@ -24,11 +25,6 @@ import {PrefetchTransform} from './prefetch';
 import {waitFor} from './streams';
 import {parsePreCacheConfig} from './sw-precache';
 
-// TODO(fks) 09-22-2016: Latest npm type declaration resolves to a non-module
```

Comment:
was there a type declaration from npm? If so, remove it. If not, then that's not what was causing the error, it was the lack of a declaration file.